### ISSUE TITLE
n8n does not execute correct workflow

### ISSUE THREAD

**Issue by enisozgen** [n8n-io/n8n#812]
Hello thanks for simple and great project. 

Few days ago I updated my n8n version from 0.46.0 to 0.75.0. It started create problem about start point of flow.

**Expected**: Application should execute PATH2 related flow.
**Current**: Application executes PATH1 and It does not care about start point.

**To Reproduce**
1. Use the json file which I shared(https://gist.github.com/enisozgen/323d8b74013523184e43fc3abfe2cb33).
2. Execute workflow.

**Environment**
 - OS: ubuntu:18.04 (dockerized)
 - n8n Version 0.75.0
 - Node.js Version: v12.18.3

**Additional context**
- I was working as expected while v 0.46.0.
- I did a little debug depends on CRON name it start the flow from different place.
  - If you change CRON1 to CRON3. It will execute first CRON2(Cares about alphabet).
  - It does not care CRON inactive if you start manually from GUI(You can see from the picture.).

**TLDR**

![tldr-8n8](https://user-images.githubusercontent.com/5619924/89156848-b9952e00-d56b-11ea-81c0-406ba3d8c5b0.jpg)

Regards

**Comment by janober** [comment]
Thanks a lot @enisozgen for making us aware of the problem.

You are right in your example above it should start from CRON2. That problem got now fixed and will be released with the next version.

**Comment by janober** [comment]
Got released with n8n@0.76.0

**Comment by enisozgen** [comment]
Hello @janober,

I have just tested same problem. Current situation is like that

![image](https://user-images.githubusercontent.com/5619924/89589992-3879b780-d847-11ea-8d6a-f2ea6857366e.png)

It does not start from start node.

Regards

**Comment by janober** [comment]
Ah yes, that is like that on purpose. As soon as there is another active trigger node in the workflow it starts automatically from that one.

**Comment by enisozgen** [comment]
Hello @janober 

I don't think that has been programmed by purpose. :) 

It's kind of a random for me now.

![picture-1](https://user-images.githubusercontent.com/5619924/89633021-760c2e00-d8a3-11ea-8b9c-150c87b26d58.png)

By changing name of the cron nodes I am able to get also this result.

![NameChange](https://user-images.githubusercontent.com/5619924/89633089-963bed00-d8a3-11ea-8267-5ef6ab9015d2.png)

It never respect start node. Even I don't tie it anywhere application starts depends on the naming of the cron node.

Another example by changing cron name without start node connection.

![pic3](https://user-images.githubusercontent.com/5619924/89634163-38a8a000-d8a5-11ea-97b0-8393dab8bd62.png)

Just change name of cron + add command node

![pic4](https://user-images.githubusercontent.com/5619924/89634408-9d63fa80-d8a5-11ea-90a3-3ca0f5f55ca5.png)

**Comment by janober** [comment]
Considering that I programmed it I am pretty sure it got programmed on purpose.

Again the logic is the following. If there is only a Start-Node it stars from there. If there is also an additional Trigger-Node it starts from there. If there are multiple Trigger-Nodes it starts from any of those (as it is impossible to know from which to start). If there is also a Webhook-Node it waits till it gets called and starts from there.

If you want it to start from a specific Trigger-Node in you have to deactivate all the other ones.

**Comment by enisozgen** [comment]
Thank you for all explanations. 👍 

Sorry for disturbing again and again.

All the best :)
